nixpkgs_pytools/prefix_modules.py

```diff
@@ -0,0 +1,302 @@
+import os
+
+if "PREFIX_PYTHON_MODULES_DEBUG" in os.environ:
+    import pdb
+
+    pdb.set_trace()
+
+import argparse
+import fnmatch
+import shutil
+import sys
+import tempfile
+import textwrap
+import traceback
+from pathlib import Path
+from typing import List, Literal, Optional, Tuple
+
+from rope.base.project import Project
+from rope.refactor.move import MoveModule
+from rope.refactor.rename import Rename
+
+parser = argparse.ArgumentParser("prefix-python-modules")
+parser.add_argument("repo_root", type=Path)
+parser.add_argument(
+    "--rename-external",
+    nargs=3,
+    action="append",
+    help="Takes (old_name, new_name, glob), e.g.:"
+    ' --rename-external utils dino.utils "**/extract_dino_features.py"',
+)
+parser.add_argument("-e", "--exclude-glob", action="append")
+parser.add_argument("--dont-catch", action="store_true")
+parser.add_argument("--prefix", required=True)
+parser.add_argument("--verbose", action="store_true")
+parser.add_argument("--quiet", action="store_true")
+parser.add_argument(
+    "--mode",
+    default="first-error",
+    choices=("first-error", "keep-going", "interactive"),
+)
+
+CATCH_ERRORS = True
+
+
+def indent(s: str, n: int = 4) -> str:
+    return textwrap.indent(s, " " * n)
+
+
+def convert_to_packages(project_root, exclude_globs: List[str]):
+    project = Project(project_root)
```

Comment:
I saw you were passing `ropefolder=None` elsewhere. I don't mind, I can do that as well for consistency